content/docs/searching.md

```diff
@@ -70,7 +70,7 @@ The various action buttons available include:
 * **Close**: Closes the search dialog
 
 * **Replace**: Replaces the currently-selected match.  (If no match is currently selected, it behaves like **Find Next** and just highlights the next match in the specified direction)
-* **Replace All**: Replaces all matches in the active document.  With **☑ Wrap Around** enabled, it makes one pass through the active document, from the very top to the very bottom.  With **☐ Wrap Around** disabled, it searches from the caret to the end of the file (if **☐ Backward direction** is disabled) or from the caret to the beginning of the file (if **☑ Backward direction** is enabled).
+* **Replace All**: With **☑ Wrap Around** enabled, it makes one pass through the active document, from the very top to the very bottom, and replaces all occurrences found.  With **☐ Wrap Around** disabled, it searches from the caret to the end of the file (if **☐ Backward direction** is disabled) or from the caret to the beginning of the file (if **☑ Backward direction** is enabled) and replaces all occurrences found in that region.
```

Comment:
Personally I don't like the "disabled"/"enabled" verbiage.  Is "unasserted"/"asserted" better?  Or "ticked"/"unticked"...  

I would change the backward-dir part to be "from the beginning of the file to the caret" since that is what is actually occurring.

Reply:
rephrased again...